src/components/totals/EntityGoalProgress/index.js

```diff
@@ -68,11 +70,17 @@ module.exports = React.createClass({
   },
 
   onSuccess: function(res) {
+    console.log(res)
+
     this.setState({
       isLoading: false,
       total: res.total_amount_cents.sum ? res.total_amount_cents.sum + this.props.offset : 0,
       goal: res.goal
     });
+
+    if (typeof this.props.onLoad === 'function') {
```

Comment:
@davidherse We usually do just check for presence, this is super defensive mind you because it protects against values of incorrect types being passed as `onLoad`. 

Unfortunately `propType`s and `defaultProp`s don't amount to a guarantee. :(

Up for discussion as to whether or not we reckon it's a good idea.